### ISSUE TITLE
Please add support for Ruby 3.1

### ISSUE THREAD

**Issue by johnnyshields** [knu/ruby-unf_ext#65]

**Comment by johnnyshields** [comment]
@knu any update on this?

**Comment by noypitt** [comment]
any updates? i am getting this errors:

```
`Gem::Ext::BuildError: ERROR: Failed to build gem native extension.

current directory:
.../ruby/3.1.0/gems/unf_ext-0.0.8/ext/unf_ext
/usr/local/bin/ruby -I /usr/local/lib/ruby/site_ruby/3.1.0 -r
./siteconf20220224-6-sny0t1.rb extconf.rb
checking for -lstdc++... no
creating Makefile

current directory: /app/worker_service/ruby/3.1.0/gems/unf_ext-0.0.8/ext/unf_ext
make DESTDIR\= clean

current directory: /app/worker_service/ruby/3.1.0/gems/unf_ext-0.0.8/ext/unf_ext
make DESTDIR\=
compiling unf.cc
make: g++: Command not found
make: *** [Makefile:214: unf.o] Error 127

make failed, exit code 2

Gem files will remain installed in
/app/worker_service/ruby/3.1.0/gems/unf_ext-0.0.8 for inspection.
Results logged to
/app/worker_service/ruby/3.1.0/extensions/x86_64-linux/3.1.0/unf_ext-0.0.8/gem_make.out

  /usr/local/lib/ruby/site_ruby/3.1.0/rubygems/ext/builder.rb:95:in `run'
/usr/local/lib/ruby/site_ruby/3.1.0/rubygems/ext/builder.rb:44:in `block in
make'
  /usr/local/lib/ruby/site_ruby/3.1.0/rubygems/ext/builder.rb:36:in `each'
  /usr/local/lib/ruby/site_ruby/3.1.0/rubygems/ext/builder.rb:36:in `make'
/usr/local/lib/ruby/site_ruby/3.1.0/rubygems/ext/ext_conf_builder.rb:63:in
`block in build'
  /usr/local/lib/ruby/3.1.0/tempfile.rb:317:in `open'
/usr/local/lib/ruby/site_ruby/3.1.0/rubygems/ext/ext_conf_builder.rb:26:in
`build'
/usr/local/lib/ruby/site_ruby/3.1.0/rubygems/ext/builder.rb:161:in
`build_extension'
/usr/local/lib/ruby/site_ruby/3.1.0/rubygems/ext/builder.rb:195:in `block in
build_extensions'
  /usr/local/lib/ruby/site_ruby/3.1.0/rubygems/ext/builder.rb:192:in `each'
/usr/local/lib/ruby/site_ruby/3.1.0/rubygems/ext/builder.rb:192:in
`build_extensions'
/usr/local/lib/ruby/site_ruby/3.1.0/rubygems/installer.rb:847:in
`build_extensions'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.7/lib/bundler/rubygems_gem_installer.rb:71:in
`build_extensions'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.7/lib/bundler/rubygems_gem_installer.rb:28:in
`install'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.7/lib/bundler/source/rubygems.rb:204:in
`install'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.7/lib/bundler/installer/gem_installer.rb:54:in
`install'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.7/lib/bundler/installer/gem_installer.rb:16:in
`install_from_spec'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.7/lib/bundler/installer/parallel_installer.rb:186:in
`do_install'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.7/lib/bundler/installer/parallel_installer.rb:177:in
`block in worker_pool'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.7/lib/bundler/worker.rb:62:in
`apply_func'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.7/lib/bundler/worker.rb:57:in
`block in process_queue'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.7/lib/bundler/worker.rb:54:in
`loop'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.7/lib/bundler/worker.rb:54:in
`process_queue'
/usr/local/lib/ruby/gems/3.1.0/gems/bundler-2.3.7/lib/bundler/worker.rb:91:in
`block (2 levels) in create_threads'

An error occurred while installing unf_ext (0.0.8), and Bundler cannot continue.
`
```

**Comment by johnnyshields** [comment]
@knu can you please handle this?

**Comment by knu** [comment]
Sorry for the delay, guys.  I just released 0.0.8.1 that should include a binary gem built for Ruby from 2.4 to 3.1.

**Comment by johnnyshields** [comment]
Thank you 🎊
